consensus/ethash/sealer_test.go

```diff
@@ -167,6 +167,8 @@ func TestRemoteMultiNotify(t *testing.T) {
 // Tests that pushing work packages fast to the miner doesn't cause any data race
 // issues in the notifications. Full pending block body / --miner.notify.full)
 func TestRemoteMultiNotifyFull(t *testing.T) {
+	t.Skip()
```

Comment:
Could you use t.SkipNow("and mention a reason of skipping here") ?

Reply:
I'll still need to use `t.Skip("reason")` as `t.Skip()` is log followed by `t.SkipNow()` call.